.licenserc.json

```diff
@@ -0,0 +1,12 @@
+{
+  "**/*.{ts,js,rs,toml}": "Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.",
+  "ignore": [
+    "testdata",
+    "cli/dts",
+    "cli/tests",
+    "std/node/tests",
+    "std/hash/_wasm/wasm.js",
+    "cli/tsc/00_typescript.js",
+    "target"
+  ]
+}
```

Comment:
While I'm strongly in favor of this PR, I'm not in favor of adding new dotfile and I'm pretty sure @ry will be of the same opinion. Is there a way to not use this file and pass arguments to the script explicitly?

Reply:
Separated the core routine of `x/license_checker` into the method `checkLicense` (ref: https://github.com/kt3k/deno_license_checker/pull/7/files ). Now this PR uses that method. The config is passed through function args, instead of using dotfiles.